docs/SOAP-Service-Methods.md

```diff
@@ -342,7 +347,17 @@ Create (initialize) a new activation for given user and application. After calli
 
 ### Method 'prepareActivation'
 
-Assure a key exchange between PowerAuth Client and PowerAuth Server and prepare the activation with given ID to be committed. Only activations in CREATED state can be prepared. After successfully calling this method, activation is in OTP_USED state.
+Assure a key exchange between PowerAuth Client and PowerAuth Server and prepare the activation with given ID to be committed. Only activations in CREATED state can be prepared.
+
+If optional `activationOtp` value is present in ECIES payload, then the value must match the OTP stored in activation's record and OTP validation mode must be ON_KEYS_EXCHANGE. 
+
+After successfully calling this method, activation is in OTP_USED or ACTIVE state, depending on the presence of an activation OTP in ECIES payload:
+
+| Situation | State after `prepareActivation`             |
+|-----------|---------------------------------------------|
+| Activation OTP is not required and is not provided | `OTP_USED` |
```

Comment:
This really looks a bit silly.

Reply:
The issue will be handled separately: https://github.com/wultra/powerauth-server/issues/396